staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -169,7 +169,7 @@ func ValidateCustomResourceDefinitionNames(names *apiextensions.CustomResourceDe
 	return allErrs
 }
 
-// specStandardValidator applies validations for different OpenAPI specfication versions.
+// specStandardValidator applies validations for different OpenAPI specification versions.
```

Comment:
This has already been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-54fdd67617b0f2ef1b5a09148f77303e. Closing.

/close